core/src/mindustry/input/InputHandler.java

```diff
@@ -241,6 +241,11 @@ public static void payloadDropped(Unit unit, float x, float y){
             pay.set(x, y);
             pay.dropLastPayload();
             pay.set(prevx, prevy);
+            pay.controlling().each(u -> {
+                if (u instanceof Payloadc p){
+                    p.dropLastPayload();
```

Comment:
This won't sync properly. You must use `Call.payloadDropped(...)` so positions are correct clientside.

Reply:
Fixed. https://github.com/Anuken/Mindustry/pull/3010/commits/12e35b643caeb1d19ffd306a884921da93fcc08a